### PR TITLE
Bug/3643 duplicate dates graphs 7 days  idea 1

### PR DESCRIPTION
**Pull request by danielgent**
## Summary

Addresses issue #3643

Fix for Analytics, AdSense, and Search Console graphs

analytics

adsense-fix

search console fix

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/adsense/components/module/ModuleOverviewWidget/Stats.js [hunk 1]
```diff
@@ -74,6 +84,7 @@ const Stats = ( {
 				color: '#616161',
 				fontSize: 12,
 			},
+			ticks: dates,
```
Comment (danielgent) [inline assets/js/modules/adsense/components/module/ModuleOverviewWidget/Stats.js:87]: This option is duplicated across the three graphs

I could actually move this logic into `GoogleChart`and have `GoogleChart` creates the `hAxis.ticks` option from its `data` prop

**Comment by danielgent** [comment]
Note my comment here detailing the visual differences https://github.com/google/site-kit-wp/issues/3643#issuecomment-882752452

File: assets/js/modules/adsense/components/module/ModuleOverviewWidget/Stats.js (hunk 1 above)
Comment (eugene-manuilov) [reply assets/js/modules/adsense/components/module/ModuleOverviewWidget/Stats.js:87]: No need to move it. It looks good to me.

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**